runestone/codelens/css/pytutor.css

```diff
@@ -221,7 +221,8 @@ div.ExecutionVisualizer #pyStdout {
   font-size: 12pt;
   padding: 4px;
   font-family: Andale mono, monospace;
-
+  color:#0a1bb688;
```

Comment:
Doesn't this just change the color for both light and dark modes?

Reply:
@bnmnetp I tried adding a CSS variable into pytutor.css to change the font color of Codelens print output. Although, it doesn't seem like this works when I check my local webpage. Maybe I have missed the mark somewhere?

![image](https://user-images.githubusercontent.com/46610441/147883075-bfe5b5d9-66fb-4a4a-b55c-609fc80ba686.png)

Reply:
Actually, disregard this previous comment, I realize that I changed the background color, not font color. However, even if this worked it would still technically be "hard coded". This leads me to have more questions: where should I be manipulating values to get the print output to be visible? Should I stay in pytutor.css, and keep working on the div in the above screenshot? Or do I need to change the CSS variables in the runestone-custom-sphinx-bootstrap.css file? Any help would be appreciated. I've screenshotted the variables for dark mode theme, which is in the bootstrap.css file. I am unsure of how to use these to make a change, because I do not see where and how they are being used in any other files.
![image](https://user-images.githubusercontent.com/46610441/147883914-261e0b2b-9dee-4a3a-89ce-c877fc930f72.png)